applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.h

```diff
@@ -196,21 +245,29 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) EmbeddedSkinVisualizationProcess :
     ///@name Member Variables
     ///@{
 
-    ModelPart&                                                                          mrModelPart;
-    ModelPart&                                                                          mrVisualizationModelPart;
+    // Unordered map to relate the newly created nodes and the origin mesh ones
+    CutNodesMapType mCutNodesMap;
+
+    // Container with the splitting newly created elements
+    ModelPart::ElementsContainerType mNewElementsPointers;
+
+    // Reference to the origin model part
+    ModelPart& mrModelPart;
 
-    CutNodesMapType                                                                     mCutNodesMap;
+    // Reference to the visualization model part
+    ModelPart& mrVisualizationModelPart;
 
-    ModelPart::ElementsContainerType                                                    mNewElementsPointers;
+    // Shape functions type. Current available options ausas and standard
+    const std::string mShapeFunctions;
```

Comment:
```suggestion
    const std::string mShapeFunctionType;
```

I would even make this an enum

Reply:
Enum done.